pseudocode.md

```diff
@@ -0,0 +1,89 @@
+# Choose Your Own Adventure!
+
```

Comment:
Awesome, pseudo code by the way! This is really great to see, and it's so neatly indented and everything :+1: